src/mainboard/sunxi/nezha/main/src/main.rs

```diff
@@ -439,8 +440,17 @@ extern "C" fn main() -> usize {
         print!("oreboot: reset reason = {}", reset_reason);
         reset_type
     } else {
+        // TODO: Do we need more stuff here?
+        unsafe {
+            asm!("csrs 0x7c0, {}", in(reg) 0x00018000);
+        }
+        print!("You are NOT MY SUPERVISOR!\r\n");
```

Comment:
in case anyone missed the ref.... https://www.youtube.com/watch?v=YEwlW5sHQ4Q

Reply:
WONDERFUL